src/main/python/ttconv/model.py

```diff
@@ -838,6 +839,18 @@ def set_active_area(self, active_area: ActiveAreaType):
 
     self._active_area = active_area
 
+  # language
+
+  def set_lang(self, language: str):
+    '''Sets the langugage of the document, as an RFC 5646 language tag.'''
```

Comment:
"langugage" instead of "language"
This typo is also present into the documentation of `Document::get_lang()` and into `ContentElement::set_lang(..)` and `ContentElement::get_lang()` :)

Reply:
Fixed at https://github.com/sandflow/ttconv/pull/55/commits/402c900c10a113d6bf72ca2b15b74d9cc9708fa0